docs/ADRs/012-tad-tool.md

```diff
@@ -0,0 +1,56 @@
+# ADR-0012 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+The Government of the Netherlands demands that government organisations are transparent about the use of
+high impact algorithms. This is a requirement of
+the [Open Government Act](https://www.rijksoverheid.nl/onderwerpen/wet-open-overheid-woo).
+This transparency can start with registering an algorithm in the
+[Algoritmeregister](https://algoritmes.overheid.nl/).
+
+Validation can be achieved by proper evaluation of algorithmic performance,
+bias and fairness and by doing assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+Transparency can be achieved by sharing these outcomes in a standardized way.
```

Comment:
I'm not sure we should start with this, this is rather the context for our team, I think we can zoom in for this tool. 

```suggestion
We are considering tooling for organisation to get more grip on their algorithms. Tooling for, for instance bias and fairness tests and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).

Transparency, we think, can be fostered by sharing reports from such a tool in a standardized way.
```